src/registration/terms-of-service/terms-of-service-alerts.tsx

```diff
@@ -0,0 +1,47 @@
+import { atom } from '@terra-ui-packages/core-utils';
+import _ from 'lodash/fp';
+import React, { useEffect } from 'react';
+import { Alert } from 'src/alerts/Alert';
+import * as Nav from 'src/libs/nav';
+import { useStore } from 'src/libs/react-utils';
+import { AuthState, authStore } from 'src/libs/state';
+
+const getNewTermsOfServiceNeedsAcceptingAlert = async (authState: AuthState) => {
+  const shouldNotify =
+    authState.termsOfService.isCurrentVersion === false && authState.termsOfService.permitsSystemUsage === true;
+  if (!shouldNotify) {
+    return null;
+  }
+
+  const message = (
+    <div key="customText">
```

Comment:
It is odd that a key is needed. Where does this component end up displaying?

Reply:
@tlangs I want to make sure you saw this question ^

Reply:
I don't think its needed, actually. This is a straight conversion from what previously existed. I'll remove the key

Reply:
Just check in the JS console that when the component is rendered, there are no warnings. Keys are necessary if things are rendered in lists, but I don't know how/where this is rendered. And often if a key is required, it is because the HTML isn't properly written.

Reply:
Removed the key, and got no warnings